HerPublicWebsite/Views/Questionnaire/CheckAnswers.cshtml

```diff
@@ -0,0 +1,206 @@
+@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@using HerPublicWebsite.BusinessLogic.Models.Enums
+@using Microsoft.AspNetCore.Mvc.TagHelpers
+@using HerPublicWebsite.Controllers
+@using System.Runtime.CompilerServices
+@using GovUkDesignSystem.Attributes
+
+@model HerPublicWebsite.Models.Questionnaire.CheckAnswersViewModel
+
+@{
+    ViewBag.Title = "Check your answers";
+}
+
+@section BeforeMain {
+    @await Html.GovUkBackLink(new BackLinkViewModel
+           {
+               Text = "Back",
+               Href = Model.BackLink,
+           })
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        
+        <form action="@Url.Action(nameof(QuestionnaireController.CheckAnswers_Post), "Questionnaire")" method="post" novalidate>
+            @(Html.HiddenFor(m => m.EntryPoint))
+            @Html.AntiForgeryToken()
+            
+            @{ var values = new Dictionary<string, object> { { "entryPoint", QuestionFlowStep.CheckAnswers } }; }
+
+            
+            <h1 class="govuk-heading-l">
+                Check your answers
+            </h1>
+            
+            <p class="govuk-body">
+                Please review the information we have gathered. You can still change your answers if anything looks wrong. 
+            </p>
+
+            @await Html.GovUkSummaryList( new SummaryListViewModel()
+                   {
+                       Rows = new List<SummaryListRowViewModel>
+                       {
+                           new()
+                           {
+                               Key = new SummaryListRowKey
+                               {
+                                   Text = "Do you have a gas boiler?"
+                               },
+                               Value = new SummaryListRowValue
+                               {
+                                   Text = Model.Questionnaire.HasGasBoiler switch{
+                                       HasGasBoiler.No => "No",
+                                       HasGasBoiler.Yes => "Yes",
+                                       HasGasBoiler.Unknown => "I don't know",
```

Comment:
This seems to duplicate the 'Checkbox label text' (as it should). Is there an easy way to reuse the same properties here, or share the logic some other way?

Failing that, I think we should still try and move this into the Enum, otherwise it'll be really easy to change the options without remembering to update this view logic.

Same applies to all the other enums below!

Reply:
There is an easy way, and I had it at first, but it felt like abusing a leaky abstraction in a way, because it's using the very specific `RadioButtonValidationAttribute`, and also didn't apply to the property ownership section as that displays different text to the options

Reply:
Yeah I think that's fair (also turns out the property ownership should be displaying the same) so maybe it is fine to use the radio checkbox label? @DanCorderSoftwire what do you think?

Reply:
It's a bit hacky as you say, but we can probably live with it.
Really we shouldn't have display attributes on that enum at all as it's business logic. But having a separate identical display enum is probably overkill.